cmd/validators.go

```diff
@@ -164,6 +165,9 @@ func InferArgumentLocation(arg string) common.Location {
 			if common.IsGCPURL(*u) {
 				return common.ELocation.GCP()
 			}
+
+			// If none of the above conditions match, return Unknown
+			return common.ELocation.Unknown()
```

Comment:
How does this impact non standard URLs?

Reply:
non standard URL wont be processed , they would follow the default case.